composer.json

```diff
@@ -4,6 +4,7 @@
     "type": "magento2-module",
     "require": {
         "magento/framework": "*",
+        "magento/module-inventory-in-store-pickup-sales": "^1.1",
```

Comment:
I do not think we should declare this dependency. Some environments might not use store pickup at all and might replace it. The plugin we define will simply do nothing if the module is not there. So I think we should not have a strict dependency on it.